validation-test/stdlib/NSNumberBridging.swift.gyb

```diff
@@ -228,8 +228,6 @@ NSNumberTests.test("_SwiftTypePreservingNSNumber(${Self}).getValue(_:), objCType
 #else
   _UnknownArchError()
 #endif
-%   elif Self == 'Bool':
-  expectedObjCType = "B"
```

Comment:
Ah, I didn't realize this was always `"c"` even on platforms where BOOL is _Bool.